xapian-applications/omega/templates/query

```diff
@@ -132,9 +141,9 @@ $SPAGE{$thispage}
 $map{$range{$add{$thispage,1},$opt{y}},$PAGE{$_}}
 $if{$ne{$opt{y},$lastpage},...}
```

Comment:
This `...` is currently naked within the `ul`, which makes it harder to restyle subsequently. You can wrap it in an `li` with a suitable classname to distinguish from the pagination controls.